### PR TITLE
Virtfs v2

### PR DESCRIPTION
**Pull request by hammerg**
An updated PR after few fixes and addressing the comments from the code review.

### CODE REVIEW

File: viofs/VirtFS Package/VirtFS Package.vcxproj.filters [hunk 1]
```diff
@@ -7,7 +7,7 @@
     </Filter>
   </ItemGroup>
   <ItemGroup>
-    <Inf Include="..\viorng\viorng.inf">
+    <Inf Include="..\pci\viofs.inf">
```
Comment (YanVugenfirer) [inline viofs/VirtFS Package/VirtFS Package.vcxproj.filters]: I think it is better to squash this commit to the initial commit fo the project.

File: viofs/VirtFS Package/VirtFS Package.vcxproj [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+  </PropertyGroup>
+  <Import Project="$(MSBuildProjectDirectory)\viofs_pack.props" />
+  <Import Project="$(MSBuildProjectDirectory)\..\..\Tools\Driver.Common.props" />
+  <ImportGroup Label="ExtensionSettings">
+  </ImportGroup>
+  <ImportGroup Label="PropertySheets">
+    <Import Project="$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props" Condition="exists('$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props')" Label="LocalAppDataPlatform" />
+  </ImportGroup>
+  <PropertyGroup Label="UserMacros" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Vista Release|Win32'">
```
[the remark this answers is not in the document]
Comment (hammerg) [reply viofs/VirtFS Package/VirtFS Package.vcxproj]: I did. I think.

File: viofs/buildAll.bat [hunk 3]
```diff
@@ -0,0 +1 @@
+@call ..\tools\build.bat viofs.sln "Wlh Win7 Win8 Win10" %*
```
[the remark this answers is not in the document]
Comment (hammerg) [reply viofs/buildAll.bat]: `>type buildAll.bat
@call ..\tools\build.bat viofs.sln "Win8 Win10" %*`

Weird.

Is the build machine have winfsp development files installed?

[the remark this answers is not in the document]
Comment (YanVugenfirer) [reply viofs/buildAll.bat]: I guess not. Please send me and Basil what needs to be installed.

**Pull request closed without merging**